plugins/inputs/smart/smart.go

```diff
@@ -112,13 +121,15 @@ var (
 )
 
 type Smart struct {
-	Path       string
-	Nocheck    string
-	Attributes bool
-	Excludes   []string
-	Devices    []string
-	UseSudo    bool
-	Timeout    internal.Duration
+	Path            string            `toml:"path"`
+	Nocheck         string            `toml:"nocheck"`
+	Attributes      bool              `toml:"attributes"`
+	ErrorCounterLog bool              `toml:"error_counter_log"`
+	Excludes        []string          `toml:"excludes"`
+	Devices         []string          `toml:"devices"`
+	UseSudo         bool              `toml:"use_sudo"`
+	Timeout         internal.Duration `toml:"timeout"`
+	Log             telegraf.Logger
```

Comment:
Could you please change to 
```suggestion
	Log             telegraf.Logger `toml:"-"`
```
so that the toml parser won't try to fill this field.